MIMIC-IV_Example/joint_script.sh

```diff
@@ -42,6 +42,9 @@ shift 4
 echo "Running pre-MEDS conversion."
 ./MIMIC-IV_Example/pre_MEDS.py raw_cohort_dir="$MIMICIV_RAW_DIR" output_dir="$MIMICIV_PREMEDS_DIR"
 
+echo "Unzipping the pre-MEDS gunzip csv files."
+for file in ${MIMICIV_PREMEDS_DIR}/*/*.csv.gz; do gzip -d --force "$file"; done
```

Comment:
actually, you probably want to do this on the raw data, right? That way the pre-meds can benefit from faster processing too?

Reply:
Done!